lib/db/mysql/index.js

```diff
@@ -27,6 +33,30 @@ function MysqlStore(options) {
     return next();
   };
   this._pool = mysql.createPool(options);
+
+  // Enforce sane defaults on every new connection.
+  // These *should* be set by the database by default, but it's nice
+  // to have an additional layer of protection here.
+  this._pool.on('connection', function(connection) {
```

Comment:
Also, ISTM that doing this logic in `_getConnection` would cause it to be run on every query, while using the pool event means it runs only when we create a new connection.